history/package.json

```diff
@@ -16,27 +16,38 @@
   "homepage": "https://cycle.js.org",
   "dependencies": {
     "@cycle/run": "*",
-    "@types/history": "4.6.x",
-    "history": "4.7.x",
+    "@types/history": "^4.6.2",
+    "history": "^4.7.2",
```

Comment:
`^4.7.2` is essentially like `4.x`, why did you decide to change this from `4.7.x`? I'm afraid third party dependencies won't follow semver that strictly

Reply:
I used ncu to update deps, will change back